test/unittests/Microsoft.ServiceFabric.AspNetCore.Tests/AspNetCoreCommunicationListenerTests.cs

```diff
@@ -32,7 +36,22 @@ public class AspNetCoreCommunicationListenerTests
         private string urlFromListenerToBuildFunc;
 
         /// <summary>
-        /// Gets or sets a value indicating whether WebHost was started/stopped.
+        /// Gets strings for the two Host Types - WebHost and GenericHost.
+        /// </summary>
+        public static IEnumerable<object[]> HostTypes
+        {
+            get
+            {
+                return new List<object[]>
```

Comment:
NIT: Enum is a better suited type

Reply:
This is xUnit specific syntax. Enum types does not work with xUnit.